content/docs/fragments.md

```diff
@@ -38,93 +38,93 @@ class Table extends React.Component {
 }
 ```
 
-`<Columns />` would need to return multiple `<td>` elements in order for the rendered HTML to be valid. If a parent div was used inside the `render()` of `<Columns />`, then the resulting HTML will be invalid.
+Komponent `<Columns />` może chcieć zwrócić kilka elementów `<td>` w celu zapewnienia poprawności wyrenderowanego kodu HTML. Gdyby komórki tabeli, zwracane przez funkcję `render()` komponentu `<Columns />`, otoczyć np. elementem `<div>`, powstały w ten sposób kod HTML byłby nieprawidłowy.
 
 ```jsx
 class Columns extends React.Component {
   render() {
     return (
       <div>
-        <td>Hello</td>
-        <td>World</td>
+        <td>Witaj</td>
+        <td>Świecie</td>
       </div>
     );
   }
 }
 ```
 
-results in a `<Table />` output of:
+daje w rezultacie następującą strukturę dla komponentu `<Table />`:
 
 ```jsx
 <table>
   <tr>
     <div>
-      <td>Hello</td>
-      <td>World</td>
+      <td>Witaj</td>
+      <td>Świecie</td>
     </div>
   </tr>
 </table>
 ```
 
-Fragments solve this problem.
+Fragmenty rozwiązują ten problem.
 
-## Usage {#usage}
+## Zasady używania {#usage}
```

Comment:
A gdyby tak po prostu
```suggestion
## Użycie {#usage}
```

Reply:
Może mam jakieś skrzywienie... :D, ale słowo "użycie" nie pasuje mi do tego kontekstu. Za to "zasady używania" (które są opisane pod tym nagłówkiem) już tak.
"Użycie" kojarzy mi się z jednym konkretnym zastosowaniem danej rzeczy, a "zasady używania" już bardziej opisują reguły postępowania.